src/lib/ui/AccessCodeForm/AccessCodeForm.tsx

```diff
@@ -80,7 +80,7 @@ function Content({
     }
 
     const start = startDate.toISO()
-    if (start == null) {
+    if (start == null || start === undefined) {
```

Comment:
I am confused how this is not redundant 🤔 Wouldn't this already have be a linter error?
![image](https://github.com/user-attachments/assets/07c9465a-687e-4bae-82ec-d71013606b8f)